content/source/docs/enterprise/api/_template.md

```diff
@@ -16,6 +16,10 @@ Parameter            | Description
 
 If further explanation of this method is needed beyond its title, write it here, after the parameter list. -->
 
+-> **Note:** This endpoint is disabled for [organization tokens](../users-teams-organizations/service-accounts.html#organization-service-accounts). You must access it with a [user token](../users-teams-organizations/users.html#api-tokens) or [team token](../users-teams-organizations/service-accounts.html#team-service-accounts).
```

Comment:
Hmm, "doesn't return data" isn't quite right as boilerplate, since we want this to also apply to things like the run endpoints that represent actions instead of data. 

I don't know, isn't responding with an error a pretty good definition of being disabled or turned off? It's technically a response, but all it says is "I'm not responding to that."

Reply:
Maybe "cannot be accessed with organization tokens"; that seems suitably generic.

Reply:
Sounds great!